whois-commons/src/main/java/net/ripe/db/whois/common/dao/ResourceDataDao.java

```diff
@@ -116,5 +116,10 @@ public int compareTo(final State other) {
                 }
             }
         }
+
+        @Override
```

Comment:
This  is a useless method - it just calls the Object.equals() which is called anyway